components/Task.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
```

Comment:
The naming of this component is a bit confusing, since it is not a single task that is displayed, but rather the whole tasklist